src/ngraph/pass/constant_folding.cpp

```diff
@@ -157,29 +198,38 @@ void pass::ConstantFolding::construct_constant_reshape()
         auto constant_match = static_pointer_cast<op::Constant>(pattern_map[constant_label]);
         auto reshape_match = static_pointer_cast<op::Reshape>(m.get_match_root());
 
+        NodeExecutorTy func = nullptr;
+        if (!m_cfmap.empty())
+        {
+            auto handler = m_cfmap.find(type_index(typeid(ngraph::op::Reshape)));
+            NGRAPH_ASSERT(handler != m_cfmap.end())
```

Comment:
I believe there are more modern versions of NGRAPH_ASSERT, @aprocter?

Reply:
Sorry, I missed this tag. `NGRAPH_ASSERT` has not been modernized yet. This is the right thing to use.